docs/manual/other-topics/migrations.md

```diff
@@ -20,7 +23,7 @@ For details see the [CLI GitHub repository](https://github.com/sequelize/cli).
 
 To create an empty project you will need to execute `init` command
 
-```text
+```sh
 npx sequelize-cli init
```

Comment:
If sequelize-cli is not installed this won't work, but I can check if it works if sequelize-cli is installed.

Reply:
Ah yes I forgot to write that part of my comment
It's only valid because we tell the user to run `yarn add sequelize-cli --dev` a couple lines above
Still it's good to double check :)